### ISSUE TITLE
Unable to disable minikube's "warning" messages

### ISSUE THREAD

**Issue by salavessa** [kubernetes/minikube#1386]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): BUG REPORT

**Minikube version** (use `minikube version`): v0.18.0 (broken since v0.17.0)

**Environment**:
- **OS** (e.g. from /etc/os-release): Windows 7 x64
- **VM Driver** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep DriverName`): virtualbox
- **ISO version** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep -i ISO` or `minikube ssh cat /etc/VERSION`): minikube-v0.18.0.iso
- **Install tools**:
- **Others**: using cygwin terminal (bash/mintty)

**What happened**: the kubectl warning message appears although the config has been updated

**What you expected to happen**: message not appear after config updated accordingly

**How to reproduce it** (as minimally and precisely as possible): run minikube in cygwin (bash or mintty)
```
$ minikube config view
========================================
kubectl could not be found on your path.  kubectl is a requirement for using minikube
To install kubectl, please do the following:

download kubectl from:
https://storage.googleapis.com/kubernetes-release/release/v1.6.0/bin/windows/amd64/kubectl.exe
Add kubectl to your system PATH

To disable this message, run the following:

minikube config set WantKubectlDownloadMsg false
========================================
- WantKubectlDownloadMsg: false
```

**Anything else do we need to know**: the message is never appearing in windows command prompt, even when the warning hasn't been disabled. I guess it's because the "warning" message goes to stderr.
As a side note, I actually have "kubectl" in the path but without the ".exe" extension, to make autocomplete work `kubectl completion bash` - so it would also be great if minikube could handle this case too.

Thanks!

**Comment by r2d4** [comment]
We should probably check for either `kubectl` or `kubectl.exe` on windows.

**Comment by r2d4** [comment]
Closed with #1397
